### ISSUE TITLE
Error when saving a webpage

### ISSUE THREAD

**Issue by enomat** [gildas-lormeau/SingleFile#631]
Error when saving a webpage

**Windows XP SP3**
**Firefox 52.9.0 ESR**
**SingleFile 1.18.53**

This (1.18.53) is the last working version in Firefox 52 ESR. But this and all older versions have this bug when I try to save this website:

1. Go to this website
https://www.gsmarena.com/

2. Click on SingleFile addon icon

Browser console output:

```
Exception { message: "", result: 2147549183, name: "NS_ERROR_UNEXPECTED", filename: "resource://gre/modules/ExtensionCon…", lineNumber: 7376, columnNumber: 0, data: null, stack: "processAttribute/<@resource://gre/m…" }  ExtensionContent.jsm:8997:7
	savePage resource://gre/modules/ExtensionContent.jsm:8997:7
Sending message that cannot be cloned. Are you trying to send an XPCOM object?  MessageChannel.jsm:547:6
Initialization error Object {  }  ui-button.js:156:5
	onMessage moz-extension://e4c681be-74e8-4a7a-beda-4195f7755bb3/extension/ui/bg/ui-button.js:156:5
	onMessage moz-extension://e4c681be-74e8-4a7a-beda-4195f7755bb3/extension/ui/bg/ui-main.js:33:12
	singlefile.extension.core.bg.messages</< moz-extension://e4c681be-74e8-4a7a-beda-4195f7755bb3/extension/core/bg/messages.js:39:11
```

**Comment by gildas-lormeau** [comment]
Closed as duplicate of #632

**Comment by gildas-lormeau** [comment]
You can now install the extension from the store: https://addons.mozilla.org/addon/singlefile-esr-52/
